### DISCUSSION TITLE
google login authentication for wax.io

### DISCUSSION THREAD

**Discussion by TechTronixx** [anonieX/alienworldsauto#8] [Ideas]
Hi,
Is possible to add google login authentication for wax.io accounts. instead of reddit.
Thanks,

**Comment by anonieX** [comment]
Hey, that is sadly not possible due to Google security. It says this when Selenium attempts to log you in:

_Couldn’t sign you in This browser or app may not be secure. 
Try using a different browser. If you’re already using a supported browser, you can refresh your screen and try again to sign in._
